packages/dev/core/src/Materials/Textures/equiRectangularCubeTexture.ts

```diff
@@ -123,7 +124,8 @@ export class EquiRectangularCubeTexture extends BaseTexture {
                     onError(`${this.getClassName()} could not be loaded`, e);
                 }
             },
-            null
+            // From ThinEngine#createTexture

```

Comment:
You should remove this comment (and the other ones) as they are only indications on how you did the PR (which is still helpful for us reviewing the PR!).